pkg/upstream/http.go

```diff
@@ -102,6 +118,15 @@ func newReverseProxy(target *url.URL, upstream options.Upstream, errorHandler Pr
 	// Inherit default transport options from Go's stdlib
 	transport := http.DefaultTransport.(*http.Transport).Clone()
 
+	if strings.HasPrefix(upstream.URI, "unix") {
```

Comment:
Can we not use the scheme at this point rather than a string prefix? We have the target right?

Reply:
I remember looking around specifically to avoid that but might have missed something

Reply:
Fixed also, I was looking mainly into upstream but the solution was in target :)